gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Copied from kotlinc
 org.gradle.jvmargs=-Duser.country=US -Dkotlin.daemon.jvm.options=-Xmx2200m -Dfile.encoding=UTF-8
 
-kotlinBaseVersion=1.4.10
-intellijVersion=193.6494.35
+kotlinBaseVersion=1.4.20
+intellijVersion=202.8194.7
```

Comment:
Can we follow kotlin-1.4.20? I.e., 201.7223.91

Reply:
https://github.com/JetBrains/kotlin/blob/master/gradle/versions.properties#L1

Reply:
done.